packages/gestalt/src/Popover/usePopover.js

```diff
@@ -0,0 +1,94 @@
+// @flow strict
+import {
+  arrow,
+  autoUpdate,
+  flip,
+  hide,
+  limitShift,
+  offset,
+  type Placement,
+  shift,
+  type Side,
+  type Strategy,
+  useFloating,
+  type UseFloatingReturn,
+} from '@floating-ui/react';
+import { type MainDirections } from '../utils/positioningTypes.js';
+
+export const DIRECTIONS_MAP: Record<MainDirections, Side | 'forceDown'> = {
```

Comment:
We need these for now to make our current `directions` work with Floating-UI's `placements`. After we roll out the chages we can remove this bridge and just use `placements` for.